samples/dotnet/kernel-syntax-examples/Example06_TemplateLanguage.cs

```diff
@@ -48,7 +49,7 @@ public static async Task RunAsync()
 
         // Show the result
         Console.WriteLine("--- Semantic Function result");
-        var result = await kindOfDay.InvokeAsync();
+        var result = await kindOfDay.SimpleInvokeAsync();
```

Comment:
These examples don't do a great job showing the benefit of this change. Can you add some new ones? I'm also concerned about the naming.

Reply:
Not sure if I follow, this example is a really good one because it this scenario there was not need to manipulate the SKContext object or get specifics (its properties)